transmogrifier/sources/ead.py

```diff
@@ -117,6 +119,80 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
 
         # format field not used in EAD
 
+        # holdings
+        for holding_element in chain(
+            collection_description.find_all("originalsloc", recursive=False),
+            collection_description.find_all("physloc", recursive=False),
```

Comment:
Per [this previous PR comment](https://github.com/MITLibraries/transmogrifier/pull/27#discussion_r963995393), `physloc` should be mapped to the `holding.location` field.